src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -247,6 +250,7 @@ public DataAccessRequest createDataAccessRequest(User user, DataAccessRequest da
           user.getEraCommonsId());
     }
     syncDataAccessRequestDatasets(datasetIds, referenceId);
+    ruleService.triggerDACRuleSettings(user, datasetIds, referenceId, request);
```

Comment:
```suggestion
    // enable this when we're ready to turn on the feature
    //ruleService.triggerDACRuleSettings(user, datasetIds, referenceId, request);
```

Reply:
We were discussing this - will try that now and check tests

Reply:
I think the risk is very low if we leave it on. There are no rules in any env and no way to enable rules via the UI (currently).